modules/uhttp/client/client.go

```diff
@@ -32,28 +32,35 @@ import (
 	"golang.org/x/net/context/ctxhttp"
 )
 
-var _serviceName string
+var (
+	_serviceName string
+	_defaultTrue = true
+)
 
 // Client wraps around a http client
 type Client struct {
 	*http.Client
-	filters []Filter
+	filters           []Filter
+	setDefaultFilters *bool
```

Comment:
This name is a bit confusing. Maybe defaultFilters added? Also add a comment indicating what it does.